src/dataframe.js

```diff
@@ -851,19 +851,33 @@ class DataFrame {
     }
 
     /**
-     * Sort DataFrame rows based on a column values. The row should contains only one variable type.
-     * @param {String} columnName The column giving order.
+     * Sort DataFrame rows based on column values. The row should contains only one variable type.
+     * @param {String | Array<string>} columnNames The columns giving order.
      * @param {Boolean} [reverse=false] Reverse mode. Reverse the order if true.
      * @returns {DataFrame} An ordered DataFrame.
      * @example
      * df.sortBy('id')
-     */
-    sortBy(columnName, reverse = false) {
+     * df.sortBy(['id1', 'id2'])
+     * df.sortBy(['id1'], true)
+    */
+    sortBy(columnNames, reverse = false) {
+        let _columnNames = columnNames;
+        if (!Array.isArray(_columnNames)) { _columnNames = [_columnNames]; }
+
+        // ensure unique columns
+        const tmp = new Set(_columnNames);
```

Comment:
In order to follow the code base, I prefer something like:
````js
const columnsToSort = Array.from(new Set(asArray(columnNames)));
````
I prefer to avoid not necessary `let` declarations.

Reply:
But it's really cool and could be applied to other functions using multiple columns as parameter.

Reply:
I was just scared about the Set capacity to keep insertion order (it wasn't the case in the past https://github.com/paulmillr/es6-shim/issues/290) but everything seems ok now.